### DISCUSSION TITLE
History of Scammer Deleted

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#112] [Ideas]
I have two methods for making a history of scammers deleted

1. History of scammers you deleted,

So suppose, there are 3 scammers on my channel:
1. Tech Support Scammer
2. MrBeast Scammer
3. ThioJoe Scammer

I then delete comments from all those 3 scammers and close the program, the next time I open the program, there will be a option, "Do you want to delete comments from the same scammers again?", if you say yes the list of the 3 scammers who I already deleted their comments will pop  up and I can pick one and then select like already there on program, either:

1. Delete comments from video
2. Delete comments from channel

And then, obviously it will automatically select the scammer they chose to delete comments from, from their history.

The next method, you can do is this,

2. Community Deleted
While I know anyone can't delete comments from any video, because this is going through the Google API, most youtubers are busy and will setup their channel mods to delete comments, and all the different people have have different devices so they can delete comments the below scammers.

1. Tech Support Scammer
2. MrBeast Scammer
3. ThioJoe Scammer

And then, when someone from another location, comes on the program, it can give them a option when they paste the video/channel url/id, "Someone with access to this channel has already deleted comments by certain scammer channels, do you want to check the list" and then if they select yes, they can see the list of scammer comments deleted on that certain video/channel by various people in the community, with access to the youtubers channel, and if they want to delete comments from the same scammer, they can select them and then, the program says:

1. Delete comments from video
2. Delete comments from channel

And then, obviously it will automatically select the scammer they chose to delete comments from, from the community.

I know this may sound confusing, @ThioJoe and may not even be possible, but just thought that this is a better way yall could do it, rather then making the user deleing comments type the same scammers name again and again.

Thanks,
Kendall Does Coding

**Comment by TechStudent10** [comment]
@KendallDoesCoding Agreed. This would be a very nice addition. Only problem, it requires setting up an API. I could probably do this ('cause I have a lot of experience in Web Development).

**Comment by KendallDoesCoding** [comment]
Yeah, I had these ideas but I'm sure it will take lots and lots of time and
coding.

**Comment by ThioJoe** [comment]
Hm i think the first one might be made redundant by the ability to ban users. But i like the idea of somehow implementing comment moderators. I know moderators have the ability to set comments as held for review, but i'm not sure if this can be done via API by moderators specifically.

However as for regular users. I believe there is a way to use the API to report comments. So i can probably add a "report only" mode that simply lets you flag comments and hope youtube does something about them.

**Reply by TechStudent10** [reply]
Yeah, a "report only" mode would definitely be cool.

**Comment by KendallDoesCoding** [comment]
On the second one, I got confused, I forgot how the API worked, I thought
if the mods login to their Google account, they automatically get access to
the channel their mod on to delete comments from there, but that's not the
case. Although, you guys can implement comment mods if you figure out how
to, but mostly the first one is good enough if you guys can implement it.
